Introduce Podman machine reset (#21684)
containers/podman: docs/source/markdown/podman-machine-reset.1.md

% podman-machine-reset 1

## NAME
podman\-machine\-reset - Reset Podman machines and environment

## SYNOPSIS
**podman machine reset** [*options*]

## DESCRIPTION

> Reset your podman machine environment.  This command stops any running machines

Reviewer: ```suggestion
Reset your Podman machine environment.  This command stops any running machines
```

and then removes them.  Configuration and data files are then removed.  Data files

> would include machine disk images and any previously pulled cache images.  When

Reviewer: A soft preference suggestion
```suggestion
can include machine disk images and any previously pulled cache images.  When
```

> this command is run, all of your Podman machines will have been deleted.

Reviewer: ```suggestion
this command is run, all of the Podman machines on the host will have been deleted.
```

or perhaps "in the environment" rather than "on the host".

## OPTIONS

#### **--force**, **-f**

Reset without confirmation.

#### **--help**

Print usage statement.

## EXAMPLES

> Reset podman machine

Reviewer: I'd just nuke this line, it feels redundant to me.

```
$ podman machine reset

> Warning: this command will delete all existing podman machines

Reviewer: ```suggestion
Warning: this command will delete all existing Podman machines
```

> and all of the configuration and data directories for podman machine

Reviewer: ```suggestion
and all of the configuration and data directories for the Podman machines.
```
